sn_networking/src/record_store.rs

```diff
@@ -202,71 +185,86 @@ impl DiskBackedRecordStore {
     }
 
     /// Prune the records in the store to ensure that we free up space
-    /// for new records.
-    ///
-    /// This can happen if we're full, or once in every X PUTs
+    /// for the incoming record.
     ///
     /// An error is returned if we prune and we're _still_ at max_record count
```

Comment:
The 'if we prune' part is incorrect. The error is returned without pruning, if I am reading correctly.